build/common.targets

```diff
@@ -267,8 +267,9 @@
       <!-- remove all net45 assemblies, otherwise loc tools have problem loading two assemblies of same name in same appdomain. net45 doesn't ship in VSIX, so doesn't need loc. -->
       <DllsToLocalizeWithMetadata Include="@(BuiltProjectOutputGroupOutput->'%(FinalOutputPath)')"
 Condition=" ('%(Extension)' == '.dll' OR '%(Filename)' == 'NuGet.CommandLine.XPlat') AND !($([System.String]::new('%(FullPath)').Contains('\net45\'))) " KeepDuplicates="false">
-        <TranslationFile>$(LocalizationRootDirectory)\{Lang}\15\%(Filename).dll.lcl</TranslationFile>    <!--Required: translation file-->
-	      <LciCommentFile>$(LocalizationRootDirectory)\comments\15\%(Filename).dll.lci</LciCommentFile>
+        <TranslationFile>$(LocalizationRepoRootDirectory)\{Lang}\15\%(Filename).dll.lcl</TranslationFile>    <!--Required: translation file-->
+        <LciCommentFile>$(LocalizationRepoRootDirectory)\comments\15\%(Filename).dll.lci</LciCommentFile>
+        <HasLceComments>false</HasLceComments>
```

Comment:
Where is this `HasLceComments` property being used? A quick search in the wiki produced [these results](https://devdiv.visualstudio.com/DevDiv/_search?text=HasLceComments%2A&type=wiki&lp=dashboard-Project&pageSize=25). The [doc](https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki/1755/MicroBuild-v1-Loc-Authoring) has following information.

If any of your projects produce an assembly which has a .resx file, you should add the following GenerateLCE property to your project file so that the comments in the .resx file are stripped out during the build and placed into a .lce file which lsbuild can use. This should match the setting in your locmanifest file for <HasLceComments>.

```
<PropertyGroup> 
  <GenerateLCE>true</GenerateLCE> 
</PropertyGroup>
```
There is a lot to learn in this area :)

Reply:
I see that you have added `Adds HasLceComments item metadata set to false on all MSBuild localization items to reduce warning count at localization time` to the PR description.

Reply:
> Where is this `HasLceComments` property being used?

That property is used in MicroBuild Localization plugin to look for `.lce` files. Those files can be used as an input to `lsbuild.exe`, specially for locking strings.

In theory, we should generate a `.lce` file for each localized assembly in our repo.

Currently, NuGet official build does not generate `.lce` files. For example See this [official build log](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=4666975&view=logs&j=2ea8c801-c4c3-578e-0ff0-bcf4c12d9404&t=c22e513a-85a0-5260-f82a-6e75295f43fd&l=89).

The root cause is: our custom target import ordering overrides a property that `GenerateLCE` targets needs to run properly.

For now, because we're not generating `.lce` files, I decided just to indicate MSBuild localization targets that we don't have .lce files

Follow-up issue [here](https://github.com/NuGet/Client.Engineering/issues/909)